test/lightning_client_mock.go

```diff
@@ -128,3 +129,20 @@ func (h *mockLightningClient) SignMessage(ctx context.Context, msg []byte) (
 	return "rdsuxrzk1dyrq3hzx6496x7tzs5f6k89cz1hecsprgssmku9q5oy6uwoxoras" +
 		"d85igaixb5chtrfxjmb8hri7tqqfqduf7etxgipec7s", nil
 }
+
+func (h *mockLightningClient) VerifyMessage(ctx context.Context, msg []byte,
+	sig string) (bool, *btcec.PublicKey, error) {
+
+	pubKeyHex := "03f5374b16f0b1f1b49101de1b9d89e0b460bc57ce9c2f9132b73df" +
+		"c76d3704daa"
```

Comment:
Same comment, should this be a mock parameter? Or assert that the sig is the same that we returned from `SignMessage` in the mock?